css/style.css

```diff
@@ -428,7 +428,7 @@ footer {
         height: 54px;
         width: 100%;
     }
-    .index-search-bar button { 
```

Comment:
Not sure where this came from, but rest assured I didn't actually touch any of this stuff.

Reply:
not changing code so no problem